src/content/docs/baas/Supabase/subapase-case-study-anasommer.md

```diff
@@ -0,0 +1,98 @@
+| Realtime Subscriptions           | Supabase supports realtime subscriptions, allowing to receive instant updates whenever data in the database changes. Subscription can be done using websockets and gives opportunity to receive real-time notofications for inserts, updates, and deletes.                                                                                                              |
+| Functions                        | Supabase provides support for serverless functions, allowing to write custom logic that runs on the server. This allows to create database functions that execute SQL queries or perform complex data operations. Additionally, Supabase offers edge functions, which allow to run serverless functions close to the users, reducing latency and improving performance. |
+| Storage                          | Supabase includes file storage capabilities, allowing to upload, store, and manage files associated with the application. The file storage integrates with popular cloud storage providers like Amazon S3 and uses Postgres to manage permissions and access control                                                                                                    |
+| AI & Vectors                     | Supabase offers an AI toolkit that allows to leverage machine learning models and embeddings in the application. Various AI-related tasks can be performed such as image recognition, natural language processing, and recommendation systems using the provided toolkit.                                                                                               |
+| Dashboard                        | Supabase provides a comprehensive dashboard that gives a centralized view of the application's data, authentication settings, APIs, functions, file storage, and more. The dashboard provides a user-friendly interface for managing and monitoring the project.                                                                                                        |
+| Example Feature                  | This feature does something for a specific reason.                                                                                                                                                                                                                                                                                                                      |
+
+## Market Comparison
+
+- Firebase and Supabase: Differ in several ways. Firebase is a NoSQL database, while Supabase is a relational database. The Firestone database, which lets to store "documents" is the core of Firebase. These are collections of key: value pairs where the value can be another sub-document. Supabase is open source. Firebase charges for reads, writes and deletes, Supabase charges based on the amount of data stored. Supabase outperforms Firebase by up to 4x on number of reads per second, and 3.1x on writes per second.
```

Comment:
```suggestion
- Firebase and Supabase: Differ in several ways. Firebase is a NoSQL database, while Supabase is a relational database. The Firestone database, which lets users store "documents" is the core of Firebase. These are collections of key: value pairs where the value can be another sub-document. Supabase is open source. Firebase charges for reads, writes and deletes, Supabase charges based on the amount of data stored. Supabase outperforms Firebase by up to 4x on number of reads per second, and 3.1x on writes per second.
```

Reply:
Maybe cite the source for this: "Supabase outperforms Firebase by up to 4x on number of reads per second, and 3.1x on writes per second."